NEWS.md

```diff
@@ -562,10 +566,10 @@ specific for admiral. Derivations like this can be implemented calling
 - Additional explanation added to `derive_param_*` and `derive_derived_param` functions regarding which variables are populated in the additional rows (#939)
 
 
-- Updated [`derive_var_worst_flag()`](https://pharmaverse.github.io/admiral/reference/derive_var_worst_flag.html) and [`derive_var_extreme_flag()`](https://pharmaverse.github.io/admiral/reference/derive_var_extreme_flag.html) vignettes to clarify their purpose (#691)
+- Updated [`derive_var_worst_flag()`](https://pharmaverse.github.io/admiral/cran-release/reference/derive_var_worst_flag.html) and [`derive_var_extreme_flag()`](https://pharmaverse.github.io/admiral/cran-release/reference/derive_var_extreme_flag.html) vignettes to clarify their purpose (#691)
```

Comment:
These links should be removed as functions are linked automatically, i.e.,  `` `derive_Var_worst_flag()` `` instead of `` [`derive_var_worst_flag()`](https://pharmaverse.github.io/admiral/cran-release/reference/derive_var_worst_flag.html)`` . Furthermore, `derive_var_worst_flag()` will be removed in 1.12 and then the link would break again.

Reply:
yes! good catch :)